src/java/org/apache/cassandra/tracing/Tracing.java

```diff
@@ -110,13 +111,14 @@ public int getTTL()
     static
     {
         Tracing tracing = null;
-        String customTracingClass = System.getProperty("cassandra.custom_tracing_class");
+        String customTracingClass = CUSTOM_TRACING_CLASS.getString();
         if (null != customTracingClass)
         {
             try
             {
                 tracing = FBUtilities.construct(customTracingClass, "Tracing");
-                logger.info("Using {} as tracing queries (as requested with -Dcassandra.custom_tracing_class)", customTracingClass);
+                logger.info("Using class {} as tracing queries (as requested with the {} system property)",
```

Comment:
imho this log message does not make, grammatically, sense. "Using class XYZ as tracing queries"? What kind of sentence is that ...

Reply:
Fixed.